src/databases/ffp/avtffpFileFormat.C

```diff
@@ -104,12 +135,12 @@ static int InitLibStripack()
 static const int dummy = InitLibStripack();
 
 static void mytrmesh(int *np, double *xp, double *yp, double *zp,
-    int *list, int *lptr, int *lend, int *lnew, int *near, int *next,
+    int *list, int *lptr, int *lend, int *lnew, int *lnear, int *next,
```

Comment:
Is this formal argument name change needed for Windows or is it merely correcting some inconsistency you found? If it is indeed *needed* for Windows, in your commit messages, can you maybe add a line indicating why the change from `near` to `lnear` was needed on Windows?

Reply:
near and far are reserved words.

Reply:
Ok, well, a common practice in Fortran/C function declaration linkages is that the pair of formal arguments such as `int *foo` and `int *lfoo` means that `lfoo` holds the length of the `foo` array. So, for a moment I was thinking you had found an inconsistency in that usage.

Since its to deal with reserved word collisions, IMHO I was suggesting it might be worth capturing that in a commit comment somehow.